libraries/AC_AttitudeControl/AC_PosControl.cpp

```diff
@@ -74,6 +74,8 @@ extern const AP_HAL::HAL& hal;
 #define POSCONTROL_VIBE_COMP_P_GAIN 0.250f
 #define POSCONTROL_VIBE_COMP_I_GAIN 0.125f
 
+AC_PosControl *AC_PosControl::_singleton;
```

Comment:
```suggestion
AC_PosControl *AC_PosControl::_singleton = nullptr;
```

This is more explicit that the pointer is null and needs to be assigned at some point. This is in alignment with what `AP_GCS` does.

Reply:
Thanks @Ryanf55 . Should I also add the `AP_HAL::panic` like in `quadplane.cpp`?

https://github.com/ArduPilot/ardupilot/blob/2b30fbcfb2b89a5a6687baf69c12c5b712d9ff7e/ArduPlane/quadplane.cpp#L615C4-L617C6

Reply:
Nvm that is what the `NO_COPY` is for. Disregard. Fixing.